src/web/screens/teamMembers.tsx

```diff
@@ -28,21 +30,61 @@ function TeamMembers() {
   const onTeamMembersUpdated = useCallback(() => {
     reloader.revalidate();
   }, [reloader]);
+  const [showStatusPopup, setShowStatusPopup] = useState<boolean>(false);
+  const [statusPopup, setStatusPopup] = useState<StatusNotificationType>();
+  const [popupDuration, setPopupDuration] = useState<number>(3000);
+
+  const handleErrorPopup = (e: Error) => {
+    const hash = Object.hasOwn(e, 'errorHash') ? `: (${(e as ApiError).errorHash})` : '';
+    setStatusPopup({
+      type: 'Error',
+      message: `${e.message}${hash}`,
+    });
+    setShowStatusPopup(true);
+    setPopupDuration(10000);
+    throw new Error(e.message);
+  };
+
+  const handleSuccessPopup = (message: string) => {
+    setStatusPopup({
+      type: 'Success',
+      message,
+    });
+    setShowStatusPopup(true);
+    setPopupDuration(3000);
+  };
 
   const handleAddTeamMember = async (formData: InviteTeamMemberForm) => {
-    await InviteTeamMember(formData, participant!.id);
+    const response = await InviteTeamMember(formData, participant!.id);
+    if (response.statusText === 'Created') {
+      handleSuccessPopup('Team member added.');
+    }
     onTeamMembersUpdated();
   };
 
   const handleRemoveTeamMember = async (userId: number) => {
-    await RemoveUser(userId);
-    onTeamMembersUpdated();
+    try {
+      const response = await RemoveUser(userId);
+      if (response.statusText === 'OK') {
```

Comment:
is there a good reason to use `response.statusText` rather than `response.status`?

Reply:
I was experimenting with sending bad requests to the API and found that if I send a request to an invalid endpoint, it returned a 200.  So I wanted to be (somewhat) sure that I was getting a successful response.  I agree this seems a little brittle and I don't love it.  I also don't love that all the statusText is 'OK' except addTeamMember which is 'Created.'  Maybe we can change that to also be 'OK'?

Reply:
I'm OK with this as-is for now, but I think we should use status codes properly - @ssundahlTTD can you write up a ticket to fix it so that we can rely on 200s to really mean success? Add details on how to reproduce the 200 on an invalid endpoint call.